fwoa-tools/.gitignore

```diff
@@ -1,3 +1,3 @@
-**migrationExport_Output.txt
```

Comment:
We should keep `.gitignore` at repository root

Reply:
deleted this gitignore and moved contents to root level gitignore